lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
```

Comment:
@jguille2 "undrop" does not work in the same way as "undo". "undrop" just recovers the latest dropped block, hence "recupera bloc", whereas "undo" undoes whatever action was perform last, hence "desfés". It is very important to keep the distinction in the translation, they are separate actions with different outcomes.